package.json

```diff
@@ -9,24 +9,24 @@
     "lib"
   ],
   "dependencies": {
-    "babel-plugin-syntax-object-rest-spread": "^6.13.0",
-    "find-up": "^2.1.0",
-    "istanbul-lib-instrument": "^1.10.1",
-    "test-exclude": "^4.2.1"
+    "@babel/plugin-syntax-object-rest-spread": "7.0.0-beta.51",
```

Comment:
@loganfsmyth, from the babel project, points out that we should revert using `@babel/plugin-syntax-object-rest-spread`; I think this major release is a good time to do so:

https://github.com/istanbuljs/babel-plugin-istanbul/pull/141#pullrequestreview-109535080

Reply:
Done.